backend/School.js

```diff
@@ -0,0 +1,83 @@
+const Class = require('./Class');
+const Student = require('./Student')
+
+class School {
+  constructor() {
+    this.classes = {
+      // className: Class Object
+        Physics: {
+          name: "Physics",
+          teacher: 'Neil de Grase Tyson',
+          students: ["Phil", "Fred"],
```

Comment:
Your `students` array should look like this:
`students: [{
       name: "Alex",
       age: 15,
       city: "Brooklyn",
       grade: 98
  },
  {
    name: "Jon",
    age: 14,
    city: "Boston",
    grade: 87
   }]

It's an array of objects where each object represents a student.